alembic/versions/18576fdffd86_add_notes_and_refund_timestamp_to_.py

```diff
@@ -0,0 +1,31 @@
+"""add_notes_and_refund_timestamp_to_transaction
+
+Revision ID: 18576fdffd86
+Revises: 4246213b032b
+Create Date: 2017-03-25 12:16:09.459952
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '18576fdffd86'
+down_revision = '4246213b032b'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    op.add_column('payment_transaction', sa.Column('internal_note', sa.Text(), nullable=True))
+    op.add_column('payment_transaction', sa.Column('refund_description', sa.Text(), nullable=True))
```

Comment:
These two should be regular string fields, not text areas, unless we're specifying a rendering format.